CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+Reporting Bugs
+# Contributing to SkillUP
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to the SkillUp Quiz project which is hosted on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Git & Github](#git-and-github)
+  * [Bootstrap 4](#bootstrap)
+  * [React](#react)
+
+[Design Descisions](#design-decisions)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Code Styleguide](#code-styleguide)
+
+[Additional Notes](#additional-notes)
+
+## Code of Conduct
+
+This project and everyone participating in it is/are maintained by the SkillUp Udacity members. By participating, you are expected to uphold this code. In case, you need to contact us, tag anyone of us in the Udacity Frontend Scholars Slack channel, below are our slack handles
+  * @ezioda004
+  * @Sunny Puri
+  * @Vanisha
+  * @kadumuri
+  * @Parul
+  * @lalitmee
+  * @Fayaz
+## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** Please don't file an issue to ask a question.You'll get faster results by using our slack handles above.
+
+
+## What should I know before I get started?
+
+### Git & Github
+This is the key thing you need to know before hopping on board, we strongly suggest you go through the below resources if you are not aware of Git & Github principles.
+  *  [Intro to Git & Github for beginners](https://medium.com/@abhishekj/an-intro-to-git-and-github-1a0e2c7e3a2f) 
+  *  [How to contribute to Open Source](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/master/CONTRIBUTING.md) 
+
+
+### Bootstrap 4
+We will be using Bootstrap 4 as the core CSS framework for this project. 
+  *  [Bootstrap 4](https://getbootstrap.com/docs/4.1/getting-started/introduction/) 
+  *  [Bootstrap 4 alternative documentation](https://www.w3schools.com/bootstrap4/default.asp) 
+  
+### React
+React JS for the behavior of this site
+  *  [React](https://reactjs.org/) 
+  * [React Tutorial](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGBuKtLgPR_zWYnrwv-JllpA)
+
+### Design Decisions
+
+We will follow a straight forward design language, its good, if you go through the current code by either forking or cloning the repository. We strongly suggest to stick with the current design guidelines.
+
+## How Can I Contribute?
+
```

Comment:
Can you add a project setup section here?
Leave it empty for now.

Everything looks great otherwise. 👍

Reply:
Done!

Reply:
Hey, @ezioda004 I will add a commit guide for PRs after merging this PR. 
Actually, that guide will be about the commit messages which we are going to put during our commit.
That will be like if you are putting a commit message, put it like this in three lines as follows
```
Summary of the commit

what the commit is doing

Fixes or closes <then url of the issue>
```
If we will commit by this method then it will take the `2nd` line as commit messge by itself which we have to put at the time of creating pull request.
In this way the commits will be more understanding and readable. :smiley: 
Tell me what do you think?

Reply:
How do you propose the commit messages should be like?

Reply:
We can follow this standard:
```
Heading or name of the commit(you can also mention file name which you are changing if it is one file)

what the commit is doing(in this you can mention that what you did like you fixed some typo or you added a functionality)

Fixes or closes <then url of the issue>(here simply you have to put any one from {Fixes or Closes} and then `url` of the issue)
```
the third line will be converted to link by itself. I think you will understand better when I will create next PR by following this method.

Reply:
@lalitmee  Alright, do so.